components/brave_wallet_ui/common/hardware/types.ts

```diff
@@ -57,8 +57,12 @@ export enum TrezorDerivationPaths {
 const DerivationSchemeTypes = [LedgerDerivationPaths.LedgerLive, LedgerDerivationPaths.Legacy, LedgerDerivationPaths.Deprecated, TrezorDerivationPaths.Default] as const
 export type HardwareDerivationScheme = typeof DerivationSchemeTypes[number]
 
+type HardwareWalletAccountBytesAddress = BraveWallet.HardwareWalletAccount & {
+  addressBytes?: Buffer
+}
+
 export type GetAccountsHardwareOperationResult = HardwareOperationResult & {
-  payload?: BraveWallet.HardwareWalletAccount[]
+  payload?: HardwareWalletAccountBytesAddress[]
```

Comment:
I created the `HardwareWalletAccountBytesAddress` type with the optional `addressBytes` field because the Solana ledger library returns address as a buffer type, not string like with Ethereum.

So the result the GetAccounts operation for Solana will be HardwareWalletAccounts where the `address : string` field is undefined, but `addressBytes : Buffer` is defined.

I don't need to change the mojom model for HardwareWalletAccount because the `address : string` property is set with the base58 encoded value prior to calling the `KeyringService::AddHardwareAccounts` mojom API to finalize.